src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp

```diff
@@ -51,15 +59,22 @@ enum Spells
     SPELL_MIND_FLAY             = 23953,
     SPELL_GREATER_HEAL          = 23954,
 
-    // Batriders Spell
+    // Bat Rider (Boss)
     SPELL_THROW_LIQUID_FIRE     = 23970,
-    SPELL_SUMMON_LIQUID_FIRE    = 23971
+    SPELL_SUMMON_LIQUID_FIRE    = 23971,
+
+    // Bat Rider (Trash)
+    SPELL_DEMO_SHOUT            = 23511,
+    SPELL_BATTLE_COMMAND        = 5115,
+    SPELL_INFECTED_BITE         = 16128,
+    SPELL_PASSIVE_THRASH        = 8876,
+    SPELL_UNSTABLE_CONCOTION    = 24024
```

Comment:
```suggestion
    SPELL_UNSTABLE_CONCOCTION    = 24024
```
Non-Blizzlike typo in this spell enum. Will need replacing across the script.